README.md

```diff
@@ -37,6 +37,7 @@
 - [Paw](https://luckymarmot.com/paw) - The ultimate REST client for Mac.
 - [PSequel](http://www.psequel.com/) - A PostgreSQL GUI tool.
 - [Sequel Pro](http://www.sequelpro.com/) - A MySQL database manager.
+- [SourceTree](https://www.sourcetreeapp.com/) - A free Git & Mercurial client for Mac & Windows.
```

Comment:
I think it would be better if the description is `A free Git & Mercurial client.` 
:)